core/types/arb_types.go

```diff
@@ -100,6 +100,13 @@ func (tx *ArbitrumUnsignedTx) setSignatureValues(chainID, v, r, s *big.Int) {
 
 }
 
+func (tx *ArbitrumUnsignedTx) effectiveGasPrice(dst *big.Int, baseFee *big.Int) *big.Int {
```

Comment:
For all these specifically I'd like  @PlasmaPower to review

Reply:
These LGTM as they match behavior with the `gasFeeCap()` and `gasTipCap()` impls which say that these txs only specify a fee cap and have a tip cap of zero. That means they'll always just pay the basefee.